providers/hns/hns_roce_u_hw_v2.c

```diff
@@ -1098,6 +1098,9 @@ static int hns_roce_u_v2_modify_qp(struct ibv_qp *qp, struct ibv_qp_attr *attr,
 
 	ret = ibv_cmd_modify_qp(qp, attr, attr_mask, &cmd, sizeof(cmd));
 
+	if (!ret)
```

Comment:
I don't know if it is correct behavior.
Sometimes you need to change QP state to be ERR if ibv_cmd_modify_qp() fails

Reply:
Sorry for being late.
If ibv_cmd_modify_qp() fails，and QP state was changed to be ERR ,  modify_qp_to_err was called again in post_send or post_recv,  it may return fails too, this may be unnecessary because hr_qp->state was just used in post_send or post_recv. 

Thanks.